torch_geometric/nn/conv/hgt_conv.py

```diff
@@ -78,23 +86,45 @@ def __init__(
 
         if not isinstance(in_channels, dict):
             in_channels = {node_type: in_channels for node_type in metadata[0]}
-
+        self.use_gmm = grouped_matmul_avail()
         self.in_channels = in_channels
         self.out_channels = out_channels
         self.heads = heads
         self.group = group
```

Comment:
Lets drop the group argument. Its not really used to group node tensors any more.